pages/events/index.vue

```diff
@@ -436,23 +458,30 @@ function toggleCode() {
       />
 
       <SectionWrapper theme="paleblue">
-        <date-filter
-          :key="dateListDateFilter"
-          :event-dates="dateListDateFilter"
-          :initial-dates="parsedInitialDates"
-          @input-selected="applyDateFilterSelectionToRouteURL"
-        />
-        <!-- Sample way to add DropdownFilter component -->
-        <!--DropdownFilter
-        :filterGroups="searchFilters"
-        :selectedFilters="userFilterSelection"
-        @input-selected="applyEventFilterSelectionToRouteURL"
-      /-->
         <TabList
           v-if="!isMobile"
           alignment="right"
           :initial-tab="parseViewSelection"
         >
+          <template #filters>
```

Comment:
I can undo it, but this is the slot that was always meant for this filters so its worrisome to me that the mobile layout is not compatible with it.

Reply:
I had to remove it totally because apparently vue components with slots mistake <!-- html comments --> for components and try to mount them, which caused page-breaking errors.